### PR TITLE
GEODE-7897: Run Redis integration tests against native Redis via Docker

### PR DESCRIPTION
**Pull request by ringles**
Configures relevant integration tests to also run against native Redis via a Docker image. Intended to help find discrepancies between Geode Redis behavior and that of native Redis.

### CODE REVIEW

**Comment by jhutchison** [comment]
re: GeoDockerAcceptanceTests- could you please create a jira ticket to implement the empty tests, if they are important to implement (or otherwise delete them)?

File: geode-redis/src/acceptanceTest/java/org/apache/geode/redis/RenameDockerAcceptanceTest.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+  public int getPort() {
+    return redisContainer.getFirstMappedPort();
+  }
+
+  @Test
+  public void testSortedSet() {
+    // TODO: rename for sorted set not currently implemented
+  }
+
```
Comment (xtreme-prasath) [inline geode-redis/src/acceptanceTest/java/org/apache/geode/redis/RenameDockerAcceptanceTest.java]: Can we please get a Jira ticket created to track these todos?

File: geode-redis/src/integrationTest/java/org/apache/geode/redis/GeoIntegrationTest.java [hunk 2]
```diff
@@ -69,9 +68,18 @@ public static void setUp() throws IOException {
 
   @After
   public void cleanup() {
+    // TODO: Correct implementation of flushAll so it deletes Geo* keys properly and this zrem is
+    // not needed
```
Comment (xtreme-prasath) [inline geode-redis/src/integrationTest/java/org/apache/geode/redis/GeoIntegrationTest.java]: Can we please create a Jira ticket to track the todos?

File: geode-redis/src/integrationTest/java/org/apache/geode/redis/GeoIntegrationTest.java [hunk 3]
```diff
@@ -80,17 +88,18 @@ public void testGeoAdd() {
...
-    String hash =
-        sicilyRegion.get(new ByteArrayWrapper(new String("Palermo").getBytes())).toString();
-    assertEquals("sqc8b49rnyte", hash);
-
-    hash = sicilyRegion.get(new ByteArrayWrapper(new String("Catania").getBytes())).toString();
-    assertEquals("sqdtr74hyu5n", hash);
+    // TODO: make sure GEOADD is tested sufficiently without needing to get the region/Geode
+    // internals and properly implemented
+    // Region<ByteArrayWrapper, ByteArrayWrapper> sicilyRegion = cache.getRegion("Sicily");
+    // assertNotNull("Expected region to be not NULL", sicilyRegion);
```
Comment (xtreme-prasath) [inline geode-redis/src/integrationTest/java/org/apache/geode/redis/GeoIntegrationTest.java]: Can we please create a Jira ticket to track the todos?

File: geode-redis/src/acceptanceTest/java/org/apache/geode/redis/RenameDockerAcceptanceTest.java (hunk 1 above)
Comment (ringles) [reply geode-redis/src/acceptanceTest/java/org/apache/geode/redis/RenameDockerAcceptanceTest.java]: Created  GEODE-7910, updated TODO comments to refer to Jira.

File: geode-redis/src/integrationTest/java/org/apache/geode/redis/GeoIntegrationTest.java (hunk 3 above)
Comment (ringles) [reply geode-redis/src/integrationTest/java/org/apache/geode/redis/GeoIntegrationTest.java]: Created  GEODE-7909, updated TODO comments to refer to Jira.

File: geode-redis/src/integrationTest/java/org/apache/geode/redis/GeoIntegrationTest.java (hunk 2 above)
Comment (ringles) [reply geode-redis/src/integrationTest/java/org/apache/geode/redis/GeoIntegrationTest.java]: Created  GEODE-7909, updated TODO comments to refer to Jira.

**Pull request merged**